.bazelci/presubmit.yml

```diff
@@ -2,37 +2,21 @@
 platforms:
   ubuntu1404:
     # TODO(jingwen): switch these to sh_test and port them to macos and windows
-    shell_commands:
-    - ./tests/integration/simple_integration_test.sh
```

Comment:
Were these setup shell scripts? Can they be deleted if they're not being used?

Reply:
Nope, these were tests from the early days of this project. I'm converting them into Java tests in #87.

Reply:
so shouldn't they stay around until #87 is in? :)

Reply:
oops. I'll get #87 ready asap.